internal/wrappers/features/run_cpp.sh

```diff
@@ -2,6 +2,11 @@
 set -e
 cd internal/wrappers
 ./build_cpp_tests.sh
-./_build_cpp_tests/KuzzleSDKStepDefs > /dev/null &
-cucumber
+./_build_cpp_tests/KuzzleSDKStepDefs &
```

Comment:
Don't forget to change this line by "./_build_cpp_tests/KuzzleSDKStepDefs > /dev/null &"
to provide cleaner CI logs